client/blocks/inline-help/style.scss

```diff
@@ -0,0 +1,237 @@
+.sidebar .sidebar__footer .button {
+	&.inline-help {
+		border-radius: 0;
+		border-left: 1px solid #c8d7e1;
+		flex: 0 1 40px;
+		margin-right: 0;
+		margin-left: auto;
+		outline: 0;
+
+		@include breakpoint( "<660px" ) {
+			flex: 0 1 56px;
+		}
+	}
+
+	&.is-active {
+		background: $blue-medium;
+
+		.gridicon {
+			fill: $white;
+		}
+	}
+}
+
+.layout.is-section-help .sidebar .inline-help {
+	background: $gray-text-min;
+	
+	.gridicon {
+		fill: $white;
+	}
+}
+
+.popover.inline-help__popover {
+	@include breakpoint( "<660px" ) {
+		position: fixed;
+			top: 47px !important;
+			right: 0 !important;
+			bottom: 0 !important;
+			left: 0 !important;
+		background: rgba( $gray-light, 0.8 );
+
+		.popover__arrow {
+			display: none;
+		}
+
+		.popover__inner {
+			top: auto !important;
+			right: auto !important;
+			bottom: auto !important;
+			left: auto !important;
+			margin: 16px;
+		}
+	}
+
+	@include breakpoint( ">660px" ) {
+		width: 320px;
+	}
+
+	&.is-top .popover__arrow:before,
+	&.is-top-left .popover__arrow:before,
+	&.is-top-right .popover__arrow:before {
+		border-top: 10px solid #f3f6f8;
+	}
+
+	.button.inline-help__button {
+		text-align: left;
+		font-weight: 400;
+		width: 100%;
+		background: $gray-light;
+		border-top: 1px solid lighten( $gray, 20% );
+		padding: 16px 16px 16px 48px;
+		font-size: 16px;
+		line-height: 1;
+		position: relative;
+		border-radius: 0 0 4px 4px;
+
+		@include breakpoint( ">660px" ) {
+			font-size: 14px;
+			padding: 12px 16px 12px 48px;
+		}
+
+		.gridicon {
+			position: absolute;
+				top: 13px;
+				left: 13px;
+
+			@include breakpoint( ">660px" ) {
+				top: 8px;
+			}
+		}
+	}
+}
+
+.inline-help__heading {
```

Comment:
Minor: iirc the coding guidelines for Calypso recommend to not indent deep using sass functionality but to list the selectors individually e.g.`.search-card .search` if the specificity is needed. 

Also, there are a lot of different absolute px values used. Wondering if some of those would benefit from storing them in variables (or deriving them from other variables) to make future maintenance/iterations easier.

Both not necessary, just thinking out loud.